fjord/alerts/models.py

```diff
@@ -69,6 +79,28 @@ def __unicode__(self):
         return self.url
 
 
+class UTCDateTimeField(fields.DateTimeField):
+    """Like DateTimeField, except it always in UTC"""
+    def to_native(self, value):
+        if value is not None and value.tzinfo is None:
+            default_tzinfo = pytz.timezone(settings.TIME_ZONE)
+            value = default_tzinfo.localize(value)
+            value = value.astimezone(pytz.utc)
+        return super(UTCDateTimeField, self).to_native(value)
+
+    def from_native(self, value):
+        result = super(UTCDateTimeField, self).from_native(value)
+        # We're using MySQL with settings.USE_TZ=False. So we need to
+        # peel off the timezone if we get sent one.
+        if result is not None and result.tzinfo is not None:
+            # First convert it to UTC
+            result = result.astimezone(pytz.UTC)
+
+            # Then drop the timezone
+            result = result.replace(tzinfo=None)
+        return result
```

Comment:
Just to clarify "I was thinking I'd store the data as UTC..." was just what I was thinking about when I wrote this, but regardless I completely forgot to match `from_native` up with `to_native`. The disconnect between the two is the crux of the problem ehre.